frontend/src/views/login/components/login-form.vue

```diff
@@ -55,16 +55,13 @@
                                         <el-dropdown-item v-if="globalStore.isIntl" command="en">
                                             English
                                         </el-dropdown-item>
-                                        <el-dropdown-item v-if="globalStore.isIntl" command="ru">
-                                            Русский
-                                        </el-dropdown-item>
                                         <el-dropdown-item command="zh">中文(简体)</el-dropdown-item>
                                         <el-dropdown-item command="tw">中文(繁體)</el-dropdown-item>
                                         <el-dropdown-item v-if="!globalStore.isIntl" command="en">
                                             English
                                         </el-dropdown-item>
                                     </el-dropdown-menu>
-                                    <el-dropdown-item v-if="!globalStore.isIntl" command="ru">Русский</el-dropdown-item>
+                                    <el-dropdown-item command="ru">Русский</el-dropdown-item>
                                 </template>
                             </el-dropdown>
                         </div>
```

Comment:
There is one potential issue in the code differences you provided:

- The `v-if` directive on the second `<el-dropdown-item>` element in the dropdown menu does not match the previous ones. Inconsistent conditions can lead to unexpected behavior.

Here's the suggested fix:

```diff
@@ -55,16 +55,13 @@
                                         <el-dropdown-item v-if="globalStore.isIntl" command="en">
                                             English
                                         </el-dropdown-item>
-<<<<<<< HEAD
-                                        <el-dropdown-item v-if="globalStore.isIntl" command="ru">
-                                            Русский
-                                        </el-dropdown-item>
-=======
+>>>>>>> c84d7b2... Update language options for international users
```

This change ensures that both sets of conditionals use the same logic (`globalStore.isIntl`) for displaying the languages.